deltacat/utils/metrics.py

```diff
@@ -128,5 +182,4 @@ def emit_timer_metrics(metrics_name, value, metrics_config, **kwargs):
         metrics_type=MetricsType.TIMER,
         metrics_config=metrics_config,
         value=value,
-        **kwargs,
```

Comment:
Why removed this? This can still be useful to pass in additional kwargs to the emit_timer_metrics itself.

Reply:
My thought back then was there would be many cases where kwargs would not be used by the emitter function. I guess that's not really that important considering the upside of more fine-grained metrics control within deltacat.